tests/units/anta_tests/routing/test_bgp.py

```diff
@@ -4357,4 +4358,104 @@
             ],
         },
     },
+    {
+        "name": "success",
+        "test": VerifyBgpRouteMaps,
+        "eos_data": [
+            {
+                "vrfs": {
+                    "default": {
+                        "peerList": [
+                            {
+                                "peerAddress": "10.100.0.8",
+                                "routeMapInbound": "RM-MLAG-PEER-IN",
+                                "routeMapOutbound": "RM-MLAG-PEER-OUT",
+                            }
+                        ]
+                    },
+                },
+            },
+        ],
+        "inputs": {
+            "bgp_peers": [{"peer_address": "10.100.0.8", "vrf": "default", "inbound_route_map": "RM-MLAG-PEER-IN", "outbound_route_map": "RM-MLAG-PEER-OUT"}]
+        },
+        "expected": {"result": "success"},
+    },
+    {
+        "name": "failure",
+        "test": VerifyBgpRouteMaps,
+        "eos_data": [
+            {
+                "vrfs": {
+                    "default": {
+                        "peerList": [
+                            {
+                                "peerAddress": "10.100.0.8",
+                                "routeMapInbound": "RM-MLAG-PEER",
+                                "routeMapOutbound": "RM-MLAG-PEER",
+                            }
+                        ]
+                    },
+                },
+            },
+        ],
+        "inputs": {
+            "bgp_peers": [{"peer_address": "10.100.0.8", "vrf": "default", "inbound_route_map": "RM-MLAG-PEER-IN", "outbound_route_map": "RM-MLAG-PEER-OUT"}]
+        },
+        "expected": {
+            "result": "failure",
+            "messages": [
+                "The following BGP peers are not configured or has an incorrect or missing route map in either the inbound or outbound direction:\n"
+                "{'10.100.0.8': {'default': {'routeMapInbound': 'RM-MLAG-PEER', 'routeMapOutbound': 'RM-MLAG-PEER'}}}"
+            ],
+        },
+    },
+    {
+        "name": "failure-inbound-map",
```

Comment:
instead of incorrect name make it for not configured route map as incorrect name already covered in upper one